### PR TITLE
Update to HtmlUnit 2.50

### PR DESCRIPTION
**Pull request by AlexanderStromer**
As requested with #34 I took it into my own hands and updated. It works fine on my side. If also ok for you, consider updating NuGet as well, thx

Had to change the tests, since wikipedia has some SSL handshake issue when requesting the page (also did not work with v2.19)

### CODE REVIEW

**Comment by asbjornu** [comment]
Awesome! 👏🏼 Could you please add a GitHub Action workflow as well, so we get some automated tests that prove everything is in working order? 🙏🏼

**Comment by AlexanderStromer** [comment]
done, also did some badge-work for the readme
let me know your thoughts

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Unit Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  runalltests:
```
Comment (asbjornu) [inline .github/workflows/unittests.yml:10]:
```suggestion
  test:
```

**Review by asbjornu — CHANGES_REQUESTED** [review]
Great stuff!

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
Thank you so much for this, @AlexanderStromer! 🙏🏼

**Pull request merged**